doc/nrf/releases/release-notes-latest.rst

```diff
@@ -129,7 +129,40 @@ The current |NCS| release is based on Zephyr v2.5.99.
 
 The following list summarizes the most important changes inherited from upstream Zephyr:
 
-* No changes yet
+* Drivers:
+
+  * Flash
+
+    * SoC embedded flash driver enabling imposes selection of
```

Comment:
I'll remove this one - it is covered by https://github.com/nrfconnect/sdk-nrf/pull/4353/